### DISCUSSION TITLE
Default Font Issue

### DISCUSSION THREAD

**Discussion by komidan** [Textualize/textual#3253] [Q&A]
The default font of Textual is not my default/preference set font for the terminal. My default is "JetBrains MONO NL", although when I launch a textual app it defaults to MS Gothic which is not a monospace font and causes some messed up visuals.

If this is a System Locale issue I apologize for the misuse of an issue report.

**Comment by TomJGooding** [comment]
Textual apps like anything else that runs in the terminal has no control over fonts.

Could you provide more detail about how you are running Textual and your environment?

It will be helpful if you run the following command and paste the results:

```
textual diagnose
```

Edit to add: Screenshots/video would also be useful!

**Reply by komidan** [reply]
I turned to using Windows Terminal and it works perfectly.

After posting this issue, one thing I did come across is realizing I've been launching CMD prompt from the run dialog box instead of the "Command Prompt" shortcut. In the past, I've noticed they are different and I should have tried other options before posting an issue. I'll still provide in case it helps in anyway.

`textual run (filename)`, and/or `python (filename)` is how I ran the file.

Textual Diagnosis:
![image](https://github.com/Textualize/textual/assets/91817886/ae01be8e-e3d3-4f56-a982-2faff6cf381c)

Difference Between the CMD prompts:
![cmdprompts](https://github.com/Textualize/textual/assets/91817886/ff7b560c-57f1-4a7c-a8d4-373189e4c9d8)

MS Gothic is a Japanese font that comes when you change system locale to Japanese, there might've been some issues when I reverted back to English locale and cmd.exe still likes the Japanese locale settings.

EDIT: Another interesting thing I came across is that when using the CMD prompt launched from run dialog box; when resizing window, cmd prompt crashes. [Link to Video Showing Crash](https://streamable.com/qymu5a)

**Reply by TomJGooding** [reply]
Glad you managed to sort it. You might be right about it being related to Windows locale somehow. Generally I think you'll get on better with Windows Terminal anyway!

I can't reproduce the cmd crash even after changing my font to MSGothic.

**Comment by komidan** [comment]
Don't launch cmd prompt from run dialog box and upgrade to Windows Terminal.
